src/index.ts

```diff
@@ -381,9 +384,10 @@ export async function cli(args: string[]) {
   }
 
   const pkgManifest = require(path.join(cwd, 'package.json'));
-  const {namedExports, webDependencies} = pkgManifest['@pika/web'] || {
+  const {namedExports, webDependencies, dedupe} = pkgManifest['@pika/web'] || {
     namedExports: undefined,
     webDependencies: undefined,
+    dedupe: []
```

Comment:
nit: lets leave this undefined, if it's not available/provided by the user. Sometimes tools like Rollup will take an explicitly empty value like `[]` to mean something different than undefined, so I'd like to future proof ourselves here, if that makes sense.

Reply:
sure - you have it tomorrow.